src/colorizer/viewport/tracks/ITrackPath.ts

```diff
@@ -0,0 +1,14 @@
+import type { TrackPathParams } from "src/colorizer/viewport/tracks/types";
+
+export interface ITrackPath {
```

Comment:
What purpose is the ITrackPath serving? Is there ever a time where you treat 2D and 3D trackpaths polymorphically?  Why do you need to enforce taht they have this common interface?